src/components/calcite-tooltip/calcite-tooltip.tsx

```diff
@@ -74,7 +74,7 @@ export class CalciteTooltip {
   /**
    * Reference HTMLElement used to position this component.
    */
-  @Prop() referenceElement!: HTMLElement | string;
+  @Prop() referenceElement: HTMLElement | string;
```

Comment:
Changing doc to not make this required since a slotted tooltip doesn't need a referenceElement.